android/app/src/main/java/com/mattermost/rnbeta/CustomPushNotification.java

```diff
@@ -244,7 +248,26 @@ protected void postNotification(int id, Notification notification) {
             ApplicationBadgeHelper.instance.setApplicationIconBadgeNumber(mContext.getApplicationContext(), CustomPushNotification.badgeCount);
         }
 
-        Notification.MessagingStyle messagingStyle = new Notification.MessagingStyle("You");
+        if (android.text.TextUtils.isEmpty(senderName)) {
+            senderName = getSenderName(senderName, channelName, bundle.getString("message"));
+        }
+
+        String personId = senderId;
+        if (!android.text.TextUtils.isEmpty(channelName)) {
+            personId = channelId;
+        }
+
+        Notification.MessagingStyle messagingStyle;
+        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.P) {
+            messagingStyle = new Notification.MessagingStyle("");
+        } else {
+            Person sender = new Person.Builder()
+                    .setKey(senderId)
+                    .setName("")
```

Comment:
I assume we're setting this to the empty string because we include the name of the sending user as part of the message

Reply:
it is being set in line 297 and 303